packages/shared/lib/core/i18n/utils/formatNumber.ts

```diff
@@ -25,7 +25,7 @@ export function formatNumber(
     const appLanguage = get(appSettings).language
 
     const formatted = Intl.NumberFormat(appLanguage, {
-        minimumFractionDigits: minDecimals ?? 2,
+        minimumFractionDigits: minDecimals ?? maxDecimals ?? 2,
```

Comment:
i think this is not 100% accurate, if maxDecimals` was 18 then the new logic means that `minimumFractionDigits` would be 18, and with the old logic it would be 2. If we are going to use maxDecimals, at least I would make a `math.min(maxDecimals,2)` to pick the lowest one. This functions is used in very critical places in the app, we should be cautious